From d2893b1d39fdaaf5e7ad8af4bbb767db1195df1a Mon Sep 17 00:00:00 2001
From: Szymon Piotr Krasuski <krasuski.szymon.piotr@gmail.com>
Date: Fri, 18 Dec 2020 16:41:23 +0100
Subject: [PATCH] Remove Polish letters from README.rst

As mentioned in Issue #70 it may be more safe to remove Polish letters also in README.rst
---
 README.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.rst b/README.rst
index b571de2..68edbfb 100644
--- a/README.rst
+++ b/README.rst
@@ -3,7 +3,7 @@ Dotty-Dict
 **********
 
 :Info: Dictionary wrapper for quick access to deeply nested keys.
-:Author: Paweł Zadrożny @pawelzny <pawel.zny@gmail.com>
+:Author: Pawel Zadrozny @pawelzny <pawel.zny@gmail.com>
 
 .. image:: https://circleci.com/gh/pawelzny/dotty_dict/tree/master.svg?style=shield&circle-token=77f51e87481f339d69ca502fdbb0c2b1a76c0369
    :target: https://circleci.com/gh/pawelzny/dotty_dict/tree/master
